packages/app/components/YearSlider/YearSlider.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { Container } from 'semantic-ui-react';
+import Slider from 'rc-slider';
+import styled from 'styled-components';
+import Routes from '../../routes';
+
+import withCurrentDate, {
+  WithCurrentDateProps
+} from '../decorators/withCurrentDate';
+
+const Wrapper = styled(Container)`
+  display: flex !important;
+  align-items: center;
+  bottom: 20px;
+  flex-direction: column;
+  position: absolute;
+`;
+
+const StyledSlider = styled(Slider)`
+  width: 600px;
+`;
+
+const handleChangeYear = (year: number) =>
+  Routes.Router.pushRoute(`/map/${year}/01/01`);
+
+const YearSlider: React.SFC<WithCurrentDateProps> = ({ currentDate }) => (
+  <Wrapper fluid>
+    <StyledSlider
+      handleStyle={{
+        borderColor: 'white',
+        height: 28,
+        width: 28,
+        marginLeft: -14,
+        marginTop: -9,
+        backgroundColor: '#4a88cb'
+      }}
+      max={2018}
+      min={0}
+      onChange={handleChangeYear}
+      railStyle={{ backgroundColor: 'grey', height: 10 }}
+      trackStyle={{ backgroundColor: '#4a88cb', height: 10 }}
+      value={currentDate.getFullYear()}
```

Comment:
I'm not sure if a slider will be the best way to select dates in the future, a lot can happen in a single year. We should probably change the Current Year card to a date picker and maybe just leave the slider as a visual for it?

Reply:
Agreed with you, I removed the slider completely.